tests/phpunit/tests/theme/customHeader.php

```diff
@@ -174,6 +174,83 @@ public function test_get_header_image_tag_with_registered_default_image_and_cust
 		$this->assertStringContainsString( sprintf( 'src="%s"', $custom ), $html );
 	}
 
+	/**
+	 * Tests default values of performance attributes for "get_header_image_tag".
+	 *
+	 * @ticket 58680
+	 */
+	public function test_get_header_image_tag_with_default_performance_attributes() {
+		$this->add_theme_support(
+			array(
+				'default-image' => 'http://localhost/default-header.jpg',
+				'width'         => 60,
+				'height'        => 60,
+			)
+		);
+
+		add_filter(
+			'wp_min_priority_img_pixels',
+			function() {
```

Comment:
```suggestion
			static function() {
```

Use static closures for minor performance improvement whenever instance access is not needed.

Reply:
Awesome, thanks for the tip @mukeshpanchal27. I've changed it in 24e883e.